agent-codeless/src/main/java/com/microsoft/applicationinsights/agentc/internal/Configuration.java

```diff
@@ -25,24 +25,42 @@
 import java.util.List;
 import java.util.Map;
 
+import com.squareup.moshi.Json;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 public class Configuration {
 
     public @Nullable String connectionString;
     public @Nullable String roleName;
     public @Nullable String roleInstance;
+    public Sampling sampling = new Sampling();
     public DistributedTracing distributedTracing = new DistributedTracing();
     public LiveMetrics liveMetrics = new LiveMetrics();
     public Map<String, String> telemetryContext = Collections.emptyMap();
     public List<JmxMetric> jmxMetrics = Collections.emptyList();
     public Map<String, Map<String, Object>> instrumentation = Collections.emptyMap();
     public List<CustomInstrumentation> customInstrumentation = Collections.emptyList();
-    public @Nullable FixedRateSampling fixedRateSampling;
 
     public boolean debug;
     public boolean developerMode;
 
+    public static class Sampling {
+
+        public @Nullable FixedRateSampling fixedRate;
```

Comment:
(I guess this PR is better suited for a rename, if needed)
Are we keeping the fixedRate terminology?

Reply:
I realized that we need to sync config with other languages, so didn't change for now, but moving under experimental to give time to sort it out.